packages/form-js-viewer/src/features/viewerCommands/ViewerCommands.js

```diff
@@ -0,0 +1,38 @@
+import UpdateFieldValidationHandler from './cmd/UpdateFieldValidationHandler';
+
+export default class ViewerCommands {
+  constructor(commandStack, eventBus) {
+    this._commandStack = commandStack;
+
+    eventBus.on('form.init', () => {
+      this.registerHandlers();
+    });
+  }
+
+  registerHandlers() {
+    Object.entries(this.getHandlers()).forEach(([ id, handler ]) => {
```

Comment:
I found the usage of CommandStack in the viewer context interesting here. Just to better understand what is happening, why did we add it? Is undo/redo a needed capability?

Reply:
CC @Skaiir